diff --git a/node_modules/svelte/transition/index.js b/node_modules/svelte/transition/index.js
index 6092b66..8827e38 100644
--- a/node_modules/svelte/transition/index.js
+++ b/node_modules/svelte/transition/index.js
@@ -44,7 +44,7 @@ function blur(node, { delay = 0, duration = 400, easing: easing$1 = easing.cubic
         css: (_t, u) => `opacity: ${target_opacity - (od * u)}; filter: ${f} blur(${u * amount}px);`
     };
 }
-function fade(node, { delay = 0, duration = 400, easing: easing$1 = easing.linear }) {
+function fade(node, { delay = 0, duration = 200, easing: easing$1 = easing.quintInOut }) {
     const o = +getComputedStyle(node).opacity;
     return {
         delay,
diff --git a/node_modules/svelte/transition/index.mjs b/node_modules/svelte/transition/index.mjs
index 3a0edc2..f5d77f1 100644
--- a/node_modules/svelte/transition/index.mjs
+++ b/node_modules/svelte/transition/index.mjs
@@ -1,4 +1,4 @@
-import { cubicInOut, linear, cubicOut } from '../easing';
+import { cubicOut, cubicInOut, quintInOut, quintOut } from '../easing';
 import { is_function, assign } from '../internal';
 
 /*! *****************************************************************************
@@ -40,7 +40,7 @@ function blur(node, { delay = 0, duration = 400, easing = cubicInOut, amount = 5
         css: (_t, u) => `opacity: ${target_opacity - (od * u)}; filter: ${f} blur(${u * amount}px);`
     };
 }
-function fade(node, { delay = 0, duration = 400, easing = linear }) {
+function fade(node, { delay = 0, duration = 200, easing = quintInOut }) {
     const o = +getComputedStyle(node).opacity;
     return {
         delay,
